src/appleseed/renderer/modeling/environmentedf/latlongmapenvironmentedf.cpp

```diff
@@ -521,6 +521,14 @@ DictionaryArray LatLongMapEnvironmentEDFFactory::get_input_metadata() const
                 Dictionary().insert("texture_instance", "Textures"))
             .insert("use", "optional")
             .insert("default", "0.0")
+            .insert("min",
```

Comment:
As discussed in my previous comment, I would suggest adding a new Exposure Multiplier scalar input with the [-64, 64] range.